docs/installation.md

```diff
@@ -38,6 +42,8 @@ $ bun upgrade
 
 {% callout %}
 **Homebrew users** — To avoid conflicts with Homebrew, use `brew upgrade bun` instead.
+
+**proto users** - Use `proto install bun --pin` instead.
```

Comment:
Bit confused here - why is `--pin` here? If I'm reading the docs correctly, re-running `proto install bun` would re-resolve the version and install the latest version right? Pinning a global default seems like an orthogonal concern.

Reply:
It's to ensure the latest version is used when the `bun` shim is executed. proto doesn't symlink the latest version as the shim, it detects the version at runtime.